schema/plugin.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "Table of Contents",
+  "description": "Table of contents settings.",
+  "properties": {
+    "collapsibleNotebooks": {
+      "type": "boolean"
```

Comment:
Ah. Sorry. I was not clear. This field should be moved to `L14`. Currently, you have a duplicate `properties` field.

Reply:
Oh oops you're right! Just fixed.